extension/src/test/suite/plots/workspace.test.ts

```diff
@@ -0,0 +1,124 @@
+import { commands } from 'vscode'
+import { afterEach, beforeEach, describe, it, suite } from 'mocha'
+import { expect } from 'chai'
+import { restore, stub } from 'sinon'
+import { buildPlots } from '../plots/util'
+import { Disposable } from '../../../extension'
+import plotsDiffFixture from '../../fixtures/plotsDiff/output'
+import { closeAllEditors } from '../util'
+import * as customPlotQuickPickUtil from '../../../plots/model/quickPick'
+import { customPlotsOrderFixture } from '../../fixtures/expShow/base/customPlots'
+import { RegisteredCommands } from '../../../commands/external'
+
+suite('Workspace Plots Test Suite', () => {
+  const disposable = Disposable.fn()
+
+  beforeEach(() => {
+    restore()
+  })
+
+  afterEach(function () {
+    this.timeout(6000)
+    disposable.dispose()
+    return closeAllEditors()
+  })
+
+  describe('dvc.views.plots.addCustomPlot', () => {
```

Comment:
Looking at other tests, `workspace.test.ts` seems to handle testing for command pallete commands. If I'm wrong though, I can move these tests inside of `index.test.ts` and delete this file!